src/main/java/au/id/micolous/metrodroid/util/Preferences.kt

```diff
@@ -149,6 +149,8 @@ actual object Preferences {
 
     actual val language: String get() = Locale.getDefault().language
 
+    actual val region: String? get() = Locale.getDefault().country
```

Comment:
We should ask `TelephonyManager` for this instead: https://github.com/micolous/metrodroid-working-copy/commit/c43bb555fadef5a192979bbb93dcbebc092ea617  This doesn't appear to require special permissions.

If your system language is set to `en_US`, you'll get US cards first, regardless of where you actually are.